docs/brand-guide.rst

```diff
@@ -0,0 +1,84 @@
+####################
+Solidity Brand Guide
+####################
+
+This brand guide features information on Solidity's brand policy and
+logo usage guidelines.
+
+The Solidity Brand
+==================
+
+The Solidity programming language is an open-source, community project
+governed by a core team. The core team is sponsored by the `Ethereum
+Foundation <https://ethereum.foundation/>`_.
+
+This document aims to provide information about how to best use the
+Solidity brand name and logo.
+
+We encourage you to read this document carefully before using the
+brand name or the logo. Your cooperation is highly appreciated!
+
+Solidity Brand Name
+===================
+
+"Solidity" should be used to refer to the Solidity programming language
+solely.
+
+Please don't use "Solidity":
```

Comment:
```suggestion
Please do not use "Solidity":
```
I think in general we should avoid contractions in the docs. Maybe we actually have a rule for that in the guideline